opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/instrument.py

```diff
@@ -219,6 +220,24 @@ def __new__(cls, *args, **kwargs):
 
 
 class Histogram(_Synchronous, APIHistogram):
+    def __init__(
+        self,
+        name: str,
+        instrumentation_scope: InstrumentationScope,
+        measurement_consumer: "opentelemetry.sdk.metrics.MeasurementConsumer",
+        unit: str = "",
+        description: str = "",
+        advisory: Optional[MetricsInstrumentAdvisory] = None,
+    ):
+        super().__init__(
+            name,
+            unit=unit,
+            description=description,
+            instrumentation_scope=instrumentation_scope,
+            measurement_consumer=measurement_consumer,
+        )
+        self.advisory = advisory
```

Comment:
This is a new part of the public API right? Should we make it protected or do users have any reason to read it?

Reply:
Made it private